website_animate/views/options.xml

```diff
@@ -16,50 +16,59 @@
-                    <we-button data-select-class="" data-name="no_animation_opt">Disable animation</we-button>
-                    <we-divider/>
-                    <we-button data-select-class="o_anim_fade_in">Fade In</we-button>
-                    <we-button data-select-class="o_anim_fade_in_down">Fade In-Down</we-button>
-                    <we-button data-select-class="o_anim_fade_in_left">Fade In-Left</we-button>
-                    <we-button data-select-class="o_anim_fade_in_right">Fade In-Right</we-button>
-                    <we-button data-select-class="o_anim_fade_in_up">Fade In-Up</we-button>
-                    <we-divider/>
-                    <we-button data-select-class="o_anim_bounce_in">Bounce In</we-button>
-                    <we-button data-select-class="o_anim_bounce_in_down">Bounce In-Down</we-button>
-                    <we-button data-select-class="o_anim_bounce_in_left">Bounce In-Left</we-button>
-                    <we-button data-select-class="o_anim_bounce_in_right">Bounce In-Right</we-button>
-                    <we-divider/>
-                    <we-button data-select-class="o_anim_rotate_in">Rotate In</we-button>
-                    <we-button data-select-class="o_anim_rotate_in_down_left">Rotate In-Down-Left</we-button>
-                    <we-button data-select-class="o_anim_rotate_in_down_right">Rotate In-Down-Right</we-button>
-                    <we-divider/>
-                    <we-button data-select-class="o_anim_zoom_in">Zoom In</we-button>
-                    <we-button data-select-class="o_anim_zoom_in_down">Zoom In-Down</we-button>
-                    <we-button data-select-class="o_anim_zoom_in_left">Zoom In-Left</we-button>
-                    <we-button data-select-class="o_anim_zoom_in_right">Zoom In-Right</we-button>
-                    <we-divider/>
-                    <we-button data-select-class="o_anim_flash">Flash</we-button>
-                    <we-button data-select-class="o_anim_pulse">Pulse</we-button>
-                    <we-button data-select-class="o_anim_shake">Shake</we-button>
-                    <we-button data-select-class="o_anim_tada">Tada</we-button>
-                    <we-divider/>
-                    <we-button data-select-class="o_anim_flip_in_x">Flip-In-X</we-button>
-                    <we-button data-select-class="o_anim_flip_in_y">Flip-In-Y</we-button>
-                </we-select>
-                <we-input string="Animation Duration" data-dependencies="!no_animation_opt"
-                          data-select-style="0.4s" data-css-property="animation-duration" data-unit="s"/>
-                <we-input string="Animation Delay" data-dependencies="!no_animation_opt"
-                          data-select-style="0s" data-css-property="animation-delay" data-unit="s"/>
-                <we-checkbox string="Each time it becomes visible" data-dependencies="!no_animation_opt"
-                             data-select-class="o_animate_both_scroll"/>
```

Comment:
What are those changes?

Reply:
Adding  `t-att-data-display-handles="handlesDisplay"` to avoid displaying 2 overlays
![6819052-9fbf32097aec9f3efecb0476cb0d8722](https://user-images.githubusercontent.com/18035234/122555035-0a096300-d03a-11eb-8e24-ea344043112e.png)

Reply:
I canceled your PR about "data-display-handles", that should not be necessary and I don't see why there would 2 trashes icons on a rebased version of master:
![Screenshot from 2021-06-18 14-22-36](https://user-images.githubusercontent.com/10338094/122560262-b3535780-d040-11eb-89d5-b08b1e3b52c4.png)

Please re-investigate why there is this bug *in your branch*

Reply:
it is not a bug in my branch, it was a bug in master and i think it is fixed.